analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -456,6 +456,13 @@ public void Method_Set_Empty(List<int> list, HashSet<int> set, Queue<int> queue,
 
 class AdvancedTests
 {
+    public void UntrackedCollections()
+    {
+        var sb = new StringBuilder();
+        sb.Clear();
+        sb.Clear();                                         // Compliant
```

Comment:
I would add a comment here on why we don't track this.
From my understanding is that we might track the string builder in the future.